src/web/common.rkt

```diff
@@ -37,40 +42,42 @@
                           ,@(if title `([title ,title]) '()))
                          ,@values)))
 
+;; TODO: TeX does not want fpcores resugared while C does. It would be great to have tex match C
+;; to make everything cleaner.
 (define languages
-  `(("TeX" . ,texify-prog)
+  `(("TeX" . ,core->tex)
     ;; TODO(interface): currently program->c doesn't take the repr into account
-    ("C" . ,(λ (prog repr) (core->c prog "code")))))
+    ("C" . ,(λ (prog) (core->c prog "code")))))
```

Comment:
I'd use `curry` here. The earlier code didn't in order to ignore the `repr` argument, which now doesn't exist.